Sources/SwiftSyntax/generated/SyntaxEnum.swift

```diff
@@ -267,7 +267,7 @@ public enum SyntaxEnum {
   #endif
   case thenStmt(ThenStmtSyntax)
   case throwStmt(ThrowStmtSyntax)
-  case thrownTypeClause(ThrownTypeClauseSyntax)
+  case throwsClause(ThrowsClauseSyntax)
```

Comment:
This is what makes this change technically API-incompatible in case you exhaustively switched over `SyntaxEnum`. The release notes should point that out. (same for `SyntaxKind`)

Reply:
`thrownTypeClause` doesn't exist in any release; it was added for typed throws, which are still experiment.

Reply:
I mean that it’s API incompatible because of the *addition* of the `throwsClause` case. Since packages aren’t built without library evolution enabled by SwiftPM, the addition of cases to enums is API-incompatible if any client exhaustively switches over them.